src/datagrid/DataGrid.stories.tsx

```diff
@@ -222,4 +219,9 @@ storiesOf("DataGrid", module)
                 style={{height: "185px"}}
             />
         </div>
+    ))
+    .add("Grid with pagination", () => (
+        <div style={{width: "80%"}}>
+            <DataGrid columns={normalColumns} rows={paginationRows.slice(0, 5)} pagination={paginationDetails} />
+        </div>
```

Comment:
Please can you add a story that has filtering, sorting and pagination?

Reply:
done